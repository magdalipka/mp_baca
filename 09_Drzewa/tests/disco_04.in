4
37
CREATE PREORDER 5 51 Adam Nowak 33 Anna Kowalska 44 Marek Mickiewicz 43 Zofia Krzak 47 Ola Nowicka
POSTORDER
PREORDER
INORDER
DEQUEMIN
DEQUEMAX
POSTORDER
NEXT 23
NEXT 43
PREV 33
PREV 47
DELETE 50
ENQUE 29 Tomasz Kowalski
DELETE 33
DELETE 47
ENQUE 20 Jan Nowak
PREV 29
NEXT 29
DELETE 20
DELETE 29
ENQUE 48 Tomasz Wolny
DELETE 33
POSTORDER
ENQUE 35 Andrzej Wolny
ENQUE 38 Andrzej Kowalski
PREV 38
NEXT 38
PREV 35
NEXT 35
ENQUE 29 Tomasz Nowak
DELETE 29
INORDER
CREATE POSTORDER 1 51 Adam Nowak
DELETE 51
CREATE POSTORDER 1 51 Adam Nowak
ENQUE 48 Tomasz Wolny
HEIGHT
6
CREATE POSTORDER 8 1 a b 2 a b 3 a b 4 a b 5 a b 6 a b 7 a b 8 a b
ENQUE 0 a b
PREORDER
POSTORDER
INORDER
HEIGHT
50
CREATE PREORDER 5 51 Adam Nowak 33 Anna Kowalska 44 Marek Mickiewicz 43 Zofia Krzak 47 Ola Nowicka
POSTORDER
PREORDER
INORDER
DEQUEMIN
DEQUEMAX
POSTORDER
NEXT 23
NEXT 43
DELETE 50
ENQUE 29 Tomasz Kowalski
POSTORDER
DELETE 33
DELETE 47
POSTORDER
ENQUE 20 Jan Nowak
POSTORDER
PREV 29
NEXT 29
DELETE 20
DELETE 29
ENQUE 48 Tomasz Wolny
DELETE 33
POSTORDER
ENQUE 35 Andrzej Wolny
ENQUE 38 Andrzej Kowalski
PREV 38
NEXT 38
PREV 35
NEXT 35
ENQUE 29 Tomasz Nowak
DELETE 29
INORDER
CREATE POSTORDER 1 51 Adam Nowak
DELETE 51
CREATE POSTORDER 1 51 Adam Nowak
ENQUE 48 Tomasz Wolny
HEIGHT
ENQUE 22 UU aa
INORDER
PREV 22
PREV 21
DELETE 22
INORDER
ENQUE 1 o k
ENQUE 99 x d
ENQUE 50 f k
POSTORDER
PREORDER
HEIGHT
28
CREATE PREORDER 10 44 Anna Kowalska 43 Bartek Bartek 35 Celina Sura 29 Daniel Golba 12 Ewa Kot 
31 Malgorzata Kowalczyk 38 Grzeorz Rokosz 48 Iwona Iwona 47 Jola Jolajna 50 Koko Jambo 51 Uga Buga
POSTORDER
PREORDER
INORDER
DEQUEMAX
DEQUEMIN
INORDER
NEXT 29
NEXT 47
PREV 51
DELETE 50
HEIGHT
DELETE 35
PREORDER
INORDER
NEXT 38
ENQUE 10 Halina Kiepska
PREV 10
DELETE 38
DELETE 31
INORDER
NEXT 29
ENQUE 38 Nowy Ktos
ENQUE 39 Jeszcze Nowszy 
NEXT 39
DEQUEMAX
DEQUEMIN
HEIGHT
