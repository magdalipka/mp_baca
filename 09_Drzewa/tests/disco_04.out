ZESTAW 1
POSTORDER: 43 - Zofia Krzak, 47 - Ola Nowicka, 44 - Marek Mickiewicz, 33 - Anna Kowalska, 51 - Adam Nowak
PREORDER: 51 - Adam Nowak, 33 - Anna Kowalska, 44 - Marek Mickiewicz, 43 - Zofia Krzak, 47 - Ola Nowicka
INORDER: 33 - Anna Kowalska, 43 - Zofia Krzak, 44 - Marek Mickiewicz, 47 - Ola Nowicka, 51 - Adam Nowak
DEQUEMIN: 33 - Anna Kowalska
DEQUEMAX: 51 - Adam Nowak
POSTORDER: 43 - Zofia Krzak, 47 - Ola Nowicka, 44 - Marek Mickiewicz
NEXT 23: BRAK
NEXT 43: 44 - Marek Mickiewicz
PREV 33: BRAK
PREV 47: 44 - Marek Mickiewicz
DELETE 50: BRAK
DELETE 33: BRAK
PREV 29: 20 - Jan Nowak
NEXT 29: 43 - Zofia Krzak
DELETE 33: BRAK
POSTORDER: 43 - Zofia Krzak, 48 - Tomasz Wolny, 44 - Marek Mickiewicz
PREV 38: 35 - Andrzej Wolny
NEXT 38: 43 - Zofia Krzak
PREV 35: BRAK
NEXT 35: 38 - Andrzej Kowalski
INORDER: 35 - Andrzej Wolny, 38 - Andrzej Kowalski, 43 - Zofia Krzak, 44 - Marek Mickiewicz, 48 - Tomasz Wolny
HEIGHT: 1
ZESTAW 2
PREORDER: 8 - a b, 7 - a b, 6 - a b, 5 - a b, 4 - a b, 3 - a b, 2 - a b, 1 - a b, 0 - a b
POSTORDER: 0 - a b, 1 - a b, 2 - a b, 3 - a b, 4 - a b, 5 - a b, 6 - a b, 7 - a b, 8 - a b
INORDER: 0 - a b, 1 - a b, 2 - a b, 3 - a b, 4 - a b, 5 - a b, 6 - a b, 7 - a b, 8 - a b
HEIGHT: 8
ZESTAW 3
POSTORDER: 43 - Zofia Krzak, 47 - Ola Nowicka, 44 - Marek Mickiewicz, 33 - Anna Kowalska, 51 - Adam Nowak
PREORDER: 51 - Adam Nowak, 33 - Anna Kowalska, 44 - Marek Mickiewicz, 43 - Zofia Krzak, 47 - Ola Nowicka
INORDER: 33 - Anna Kowalska, 43 - Zofia Krzak, 44 - Marek Mickiewicz, 47 - Ola Nowicka, 51 - Adam Nowak
DEQUEMIN: 33 - Anna Kowalska
DEQUEMAX: 51 - Adam Nowak
POSTORDER: 43 - Zofia Krzak, 47 - Ola Nowicka, 44 - Marek Mickiewicz
NEXT 23: BRAK
NEXT 43: 44 - Marek Mickiewicz
DELETE 50: BRAK
POSTORDER: 29 - Tomasz Kowalski, 43 - Zofia Krzak, 47 - Ola Nowicka, 44 - Marek Mickiewicz
DELETE 33: BRAK
POSTORDER: 29 - Tomasz Kowalski, 43 - Zofia Krzak, 44 - Marek Mickiewicz
POSTORDER: 20 - Jan Nowak, 29 - Tomasz Kowalski, 43 - Zofia Krzak, 44 - Marek Mickiewicz
PREV 29: 20 - Jan Nowak
NEXT 29: 43 - Zofia Krzak
DELETE 33: BRAK
POSTORDER: 43 - Zofia Krzak, 48 - Tomasz Wolny, 44 - Marek Mickiewicz
PREV 38: 35 - Andrzej Wolny
NEXT 38: 43 - Zofia Krzak
PREV 35: BRAK
NEXT 35: 38 - Andrzej Kowalski
INORDER: 35 - Andrzej Wolny, 38 - Andrzej Kowalski, 43 - Zofia Krzak, 44 - Marek Mickiewicz, 48 - Tomasz Wolny
HEIGHT: 1
INORDER: 22 - UU aa, 48 - Tomasz Wolny, 51 - Adam Nowak
PREV 22: BRAK
PREV 21: BRAK
INORDER: 48 - Tomasz Wolny, 51 - Adam Nowak
POSTORDER: 1 - o k, 50 - f k, 48 - Tomasz Wolny, 99 - x d, 51 - Adam Nowak
PREORDER: 51 - Adam Nowak, 48 - Tomasz Wolny, 1 - o k, 50 - f k, 99 - x d
HEIGHT: 2
ZESTAW 4
POSTORDER: 12 - Ewa Kot, 31 - Malgorzata Kowalczyk, 29 - Daniel Golba, 38 - Grzeorz Rokosz, 35 - Celina Sura, 43 - Bartek Bartek, 47 - Jola Jolajna, 50 - Koko Jambo, 48 - Iwona Iwona, 44 - Anna Kowalska
PREORDER: 44 - Anna Kowalska, 43 - Bartek Bartek, 35 - Celina Sura, 29 - Daniel Golba, 12 - Ewa Kot, 31 - Malgorzata Kowalczyk, 38 - Grzeorz Rokosz, 48 - Iwona Iwona, 47 - Jola Jolajna, 50 - Koko Jambo
INORDER: 12 - Ewa Kot, 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 35 - Celina Sura, 38 - Grzeorz Rokosz, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona, 50 - Koko Jambo
DEQUEMAX: 50 - Koko Jambo
DEQUEMIN: 12 - Ewa Kot
INORDER: 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 35 - Celina Sura, 38 - Grzeorz Rokosz, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona
NEXT 29: 31 - Malgorzata Kowalczyk
NEXT 47: 48 - Iwona Iwona
PREV 51: BRAK
DELETE 50: BRAK
HEIGHT: 4
PREORDER: 44 - Anna Kowalska, 43 - Bartek Bartek, 38 - Grzeorz Rokosz, 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 48 - Iwona Iwona, 47 - Jola Jolajna
INORDER: 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 38 - Grzeorz Rokosz, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona
NEXT 38: 43 - Bartek Bartek
PREV 10: BRAK
INORDER: 10 - Halina Kiepska, 29 - Daniel Golba, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona
NEXT 29: 43 - Bartek Bartek
NEXT 39: 43 - Bartek Bartek
DEQUEMAX: 48 - Iwona Iwona
DEQUEMIN: 10 - Halina Kiepska
HEIGHT: 4